speech/google/cloud/speech/operation.py

```diff
@@ -123,8 +123,9 @@ def _update(self, response):
         raw_results = response.get('response', {}).get('results', None)
```

Comment:
Future PR: Let's collapse some / all of this into `google.cloud.operation.Operation`. This class is for the HTTP version of a long-running operation, but it's still the exact same proto: https://github.com/googleapis/googleapis/blob/050400df0fdb16f63b63e9dee53819044bffc857/google/longrunning/operations.proto#L80

It should be fine since it's already a subclass?